config/settings/default.yml

```diff
@@ -5,12 +5,19 @@ common:
 xdomain_session_timeout: 5000
 auto_pageview_timeout: 1000
 send_auto_pageview: true
+custom_tlds:
+  - skroutzstore.gr
 cookies:
   first_party_enabled: true
   version: 1
   analytics:
     name: "__sa_session"
     duration: 1800 # 30 minutes
+  session:
+    name: "__skranal_session"
+    duration: 60*60*24*30 # 30 days
```

Comment:
[Minor] duration: 2592000 # 30 days

Reply:
I think it's easier to change this way..